@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf

```diff
@@ -169,7 +169,8 @@ while {
     alive _heli
 }
 do {
-    private _battBusOn       = _heli getVariable "fza_systems_battBusOn";
+    private _battBusOn              = _heli getVariable "fza_systems_battBusOn";
+    ace_map_vehicleLightCondition   = {isLightOn [_vehicle, [0]]}; 
```

Comment:
Could this be set on a `getIn` event handler instead?

Reply:
https://github.com/AH-64D-Apache-Official-Project/AH-64D/blob/master/%40AH-64D%20Apache%20Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventGetIn.sqf

Reply:
the flood lighting one has to be in a while loop to work
i wish it would work in a get in but it dosent

Reply:
```sqf
compile format ["(driver _vehicle == _unit) || {gunner _vehicle == _unit} || {%1}", _vehicleLightCondition];
```

Reply:
Understood, thanks for answer. Approving